presto-docs/src/main/sphinx/admin/tuning.rst

```diff
@@ -10,29 +10,392 @@ Config Properties
++  ``task.operator-pre-allocated-memory``  +          ``String`` (data size)          +             ``16 MB``              +Memory preallocated for each driver in query           +
++                                          +                                          +                                    +execution. Increasing this value may cause less        +
++                                          +                                          +                                    +efficient memory usage but allows to fail fast in low  +
++                                          +                                          +                                    +memory environment more frequently.                    +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++       ``task.share-index-loading``       +               ``Boolean``                +             ``false``              +It allows to control whether index lookups join has    +
++                                          +                                          +                                    +index shared within a task. This enables the           +
++                                          +                                          +                                    +possibility of optimizing for index cache hits or for  +
++                                          +                                          +                                    +more CPU parallelism depending on the property value.  +
++                                          +                                          +                                    +Serves as default for ``task_share_index_loading``     +
++                                          +                                          +                                    +session property.                                      +
++------------------------------------------+------------------------------------------+------------------------------------+-------------------------------------------------------+
++          ``task.writer-count``           +               ``Integer``                +               ``1``                +Defines default number of writers per task. Serves as  +
++                                          +                                          +                                    +default for session property ``task_writer_count``.    +
++                                          +                                          +                                    +Setting this value to higher than default may increase +
++                                          +                                          +                                    +write speed especially in high IOPS, low write speed   +
```

Comment:
I would drop everything after "speed especially" and replace it with:

"speed especially when query is NOT IO bounded and could use of more CPU cores for parallel writes."